akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/SerializationSupport.scala

```diff
@@ -25,6 +25,7 @@ import akka.util.ccompat._
 /**
  * Some useful serialization helper methods.
  */
+@ccompatUsedUntil213
```

Comment:
I think it would be good if you can explain in standup (next week) how this works and what to watch out for when reviewing PRs

Reply:
Sure - it doesn't really do anything apart from documenting ccompat is only intended be used until 2.13 in this class, and get rid of the 'unused import' warning on 2.13.